build.gradle

```diff
@@ -70,6 +72,12 @@ publishing {
     }
 }
 
+jar {
```

Comment:
Does this work with jar? 
Is the implementation-version present in the manifest of the built jar file?

If it doesn't work, then bootJar should be used instead.

Reply:
This one doesn't. There is another entry in the `bootJar` section that works. I've moved everything manifest-related to `bootJar` for clarity.